aws/load-balancer.tf

```diff
@@ -43,6 +43,27 @@ resource "aws_lb_listener" "rancher_https" {
   }
 }
 
+resource "aws_s3_bucket" "rancher_lb_access_logs" {
+  count         = "${var.rancher_lb_access_logs_bucket_create ? 1 : 0}"
+  bucket        = "${var.rancher_lb_access_logs_bucket}"
+  acl           = "private"
+  force_destroy = true
+
+  lifecycle_rule {
+    enabled = true
+
+    expiration {
+      days = 10
+    }
+  }
+
+  tags {
+    Name        = "Access logs for Rancher loadbalancer"
+    Environment = "${var.environment}"
+    env         = "${var.environment}"
```

Comment:
Name is usually a noun. `Description` is probably what you are looking for.
You may want to use something like the following so as to get default tags being passed in which include env and Environment.
`tags = "${merge(map("Name", "rancher"), var.cloud_tags)}"`